ynput/OpenPype
# Merge feature from #4793 and #4996 to set frame range on instances

## Changelog Description
New PR that brings together the work of:
https://github.com/ynput/OpenPype/pull/4793
https://github.com/ynput/OpenPype/pull/4996
The two PR is no longer functional, it now works with the latest update_frame_range changes.

## Testing notes:
1. Open Maya Scene with wrong frame range in instances
2. Click `set frame range` button to solve this problem
Or 
1. Build scene with `Build workfile template`
2. See all instances with the right frame range
